File: server/routes/jwks.js
"use strict";

const express = require("express");
const router  = express.Router();
const { jwks } = require("../lib/keys");

router.get("/.well-known/jwks.json", (req, res) => {
  res.setHeader("Cache-Control", "public, max-age=3600");
  res.json(jwks);
});

module.exports = router;

File: internal/handler/jwks.go
package handler

import "net/http"

// GET /.well-known/jwks.json
func (a *App) JWKS(w http.ResponseWriter, r *http.Request) {
	w.Header().Set("Cache-Control", "public, max-age=3600")
	jsonOK(w, a.Keys.JWKS())
}
